cypress/e2e/supervision/report/preselect-for-review.cy.js

```diff
@@ -0,0 +1,51 @@
+beforeEach(() => {
+  cy.loginAs("Case Manager");
+  cy.createClient()
+    .withOrder()
+    .withSupervisionLevel()
+    .withActiveOrderStatus()
+  cy.get("@order").then(({id: orderId}) => {
+    cy.get("@client").then(({id: clientId}) => {
+      cy.visit(`/supervision/#/clients/${clientId}?order=${orderId}`);
+    });
+  });
+  cy.get(".TABS_REPORTS").click();
+});
+  describe(
+  "Preselect report for review",
+  { tags: ["@supervision", "@reports", "@preselect-report-for-review"] },
+  () => {
+    it("Successfully preselect a report for review", () => {
+      cy.contains('View pending report')
+      cy.wait(1000) // sad times :(
```

Comment:
I appreciate you might have tried this but I had some success with the `cy.get('input[name="firstName"]', { timeout: 2000 }).should('be.visible')`. Would that work here?

Reply:
Hmm I don't think so... the element is there but Angular does a random page reload halfway through my assertions causing the test to break. To complicate things further, the assertions are within the "Pending report" expandable box that is closed on page load. So I'm basically having to check that the page has loaded, wait a second for Angular to reload the page, then expand the pending report window and do the assertions.

Reply:
Not something I've tried... do you mean a `cy.reload()`? Angular would still want to do its own reload after the reload 🤔

Reply:
Yeah I did, ok thought it was worth asking just in case